HCIDOCS-689-413: Fix OCP landing page 4.13 (#91972)
openshift/openshift-docs: modules/assisted-installer-using-the-assisted-installer.adoc

[id="using-the-assisted-installer_{context}"]
= Using the Assisted Installer

> The {ai-full} is a user-friendly installation solution offered on the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[Red Hat Hybrid Cloud Console]. The {ai-full} reduces installation complexity, enabling users to deploy and configure {product-title} clusters with minimal manual intervention.

Reviewer: ```suggestion
The {ai-full} is a user-friendly installation solution offered on the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[Red{nbsp}Hat Hybrid Cloud Console]. The {ai-full} reduces installation complexity, so that you can deploy and configure {product-title} clusters with minimal manual intervention.
```

Reviewer: https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

![Uploading Screenshot From 2025-04-10 10-08-28.png…]()

> See the link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform[Assisted Installer for OpenShift Container Platform] documentation for details.

Reviewer: ```suggestion
See the link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform[Assisted Installer for {product-title}] documentation for details.
```
